semaphoreci.sh

```diff
@@ -93,7 +93,7 @@ setup() {
 
     ## Configure GCC to build a D compiler.
     ${SEMAPHORE_PROJECT_DIR}/configure --enable-languages=c++,d,lto --enable-checking \
-        --enable-link-mutex --disable-bootstrap --disable-libgomp --disable-libmudflap \
+        --enable-link-mutex --disable-libgomp --disable-libmudflap \
```

Comment:
IMO, if we're going to do bootstrap builds, you should also configure with `--enable-checking=release` to make sure that compilation is not needlessly slowed down.

Perhaps this could be split into two functions: `configure_test()` and `configure_release()`

Reply:
Sure, right now I just wanted to *cough* abuse the semaphore ci infrastructure to test whether there's something wrong with my build system.